src/libs/relay/CMakeLists.txt

```diff
@@ -218,6 +218,9 @@ if(ZFP_FOUND)
     list(APPEND conduit_relay_deps zfp)
 endif()
 
+if (WIN32)
+  list(APPEND conduit_relay_deps ws2_32)
```

Comment:
Looks like civetweb needs this dep for the minigw case?

Is there anyway we can detect this case more specifically, I am not sure we need it for general builds.

Reply:
ok, it seems to be needed only for mingw, I modified the code to use "if (MINGW)"